src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -1685,6 +1685,15 @@ protected override void EnterParameter(ParameterSymbol parameter)
                 if (slot > 0) SetSlotState(slot, true);
                 NoteWrite(parameter, value: null, read: true);
             }
+
+            // Mark attribute arguments as used.
+            foreach (var attrData in parameter.GetAttributes())
```

Comment:
>foreach (var attrData in parameter.GetAttributes()) [](http://example.com/codeflow?start=12&length=51)

I think we should use the same approach as for parameter default values. Since it is already in main, consider merging that code into this branch.

Reply:
> Since it is already in main, consider merging that code into this branch.

It's not in `main`, only in `features/lambda-default-parameters`.

> I think we should use the same approach as for parameter default values.

That was my original intention; then I thought that I would reuse the already-bound attribute instead of binding it again. But I now see how that's problematic. What about this? c40dc7380561f8c88baa89975ba118a047b97621